post.md

```diff
@@ -1,22 +1,22 @@
 For a while I've wanted to explore Suspense.
-I read a react blog post way back in 2019 about
+I read a React blog post way back in 2019 about
 [building great UX with React Suspense](https://legacy.reactjs.org/blog/2019/11/06/building-great-user-experiences-with-concurrent-mode-and-suspense.html).
-I remember it giving me a real aha moment.
-The description of the waterfall approach to loading data vs. loading data as you render really resonated with performance problems we'd had in a project I was working with at the time.
-Suspense though was experimental in react 16; the only thing that it was supported with was `Lazy` components,
-and I never like using experimental API's in production
+I remember it giving me a real "aha!" moment.
+The description of the waterfall approach to loading data vs loading data as you render really resonated due to performance problems we'd had in a project I was working with at the time.
+Suspense though was experimental in React 16; the only thing that it was supported with was `Lazy` components,
+and I never like using experimental APIs in production
 so I eagerly awaited the arrival of a stable API.
 
 But that day never arrived. I was put off and lost interest.
 
 Years later the react team announced some
 really [interesting](https://react.dev/blog/2022/03/29/react-v18#new-feature-transitions)
 [features](https://react.dev/blog/2022/03/29/react-v18#new-suspense-features) in v18
-(concurrent rendering - multiverse theory for react applications?).
+(concurrent rendering - multiverse theory for React applications?).
 
 But while Suspense is now "Stable" and "Production Ready" it is _still_ only recommended for use in
```

Comment:
you say "still" here, but then below it seems a bit of a duplication iwth the quote about data fetching libraries and the quote is a bit clearer. I'd combine the two with the comment about "Stable" and "Production Ready" but with the warning about data fetching libraries